docs/topics/cache.txt

```diff
@@ -195,6 +195,13 @@ In this example, the cache table's name is ``my_cache_table``::
         }
     }
 
+Unlike other cache backends, the database cache does not support automatic
+culling of expired entries at the database level. Instead, expired cache
+entries are culled each time ``set()``, ``add()``, or ``touch()`` is
+called on a particular key. If you are using the database cache backend, you
+may want to cull entries manually using a script.
+
+
```

Comment:
All expired entries are removed not only for a particular key, see https://github.com/django/django/blob/fa4e963ee7e6876581b5432363603571839ba00c/django/core/cache/backends/db.py#L250-L270
Also, I don't think there a need for using a script.
```suggestion
called.
```

Reply:
Oh wow, I totally misunderstood, thank you for being so right!